### PR TITLE
The Cook now only works under siege.

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball
fix: The Cook now ONLY works under siege.
/:cl:

### CODE REVIEW

**Comment by Shadowflame909** [comment]
one of the areas should also be botany, because cooks have access there as well!

**Comment by Buggy123** [comment]
iNB4 robust cooks steal the blueprint roundstart and expand the bar to the entire station.

**Comment by Mey-Ha-Zah** [comment]
Goofball we know you like cooks.

BUT STOP GIVING THEM SUPERPOWERS!!

**Comment by MrStonedOne** [comment]
Dupe of #35562

**Comment by MMMiracles** [comment]
Make it so having war ops/clock cult declared on a station allows him to use CQC anywhere on-station.

**Comment by Iamgoofball** [comment]
incorrect @MrStonedOne this is a different approach to the problem

his PR ties it to clothes, which doesn't fix the issue of cooks joining security roundstart

this ties it to area which does fix the problem

**Comment by Iamgoofball** [comment]
@MMMiracles not in clockcult, that's station sieging the clockcult base

**Comment by MMMiracles** [comment]
War clocks has them declare on the station, though. Which is why he only has the CQC while on-station.

**Comment by Xhuis** [comment]
You should probably make sure they don't lose their inability to use unarmed in general while they're outside of these areas. My idea for it was making it an unremovable implant that removed the martial art from them when they left those areas and gave it to them when they entered them.

**Comment by Iamgoofball** [comment]
@Xhuis Returning 0 on the martial art harm/disarm/grab overrides makes it default to stock.

**Comment by XDTM** [comment]
is it possible to define the station as the kitchen with blueprints?

**Comment by MrStonedOne** [comment]
In general, anything that requires snowflaking shit in a way that would break a "build a station" event i don't like.

One of the appeals of ss13 is that the entire world can be destroyed and remade, snowflaking areas breaks that.

Make it be based off of proximity to the oven or fridge or deep fryer or some shit.

**Comment by Shadowflame909** [comment]
well if its based on any area named "Kitchen" then if the kitchen were to be exploded, and need to be rebuilt. Couldn't the cook just request help from the CE?

**Comment by Iamgoofball** [comment]
We should instead let permits/blueprints allow you to "set" a base area type

File: code/datums/martial/cqc.dm [hunk 1]
```diff
@@ -8,6 +8,14 @@
 	name = "CQC"
 	help_verb = /mob/living/carbon/human/proc/CQC_help
 	block_chance = 75
+	var/just_a_cook = FALSE
+	var/static/list/areas_under_siege = typecacheof(list(/area/crew_quarters/kitchen,
+														/area/crew_quarters/cafeteria,
+														/area/crew_quarters/bar))
+
+/datum/martial_art/cqc/under_siege
+	name = "Culinary School Training"
```
Comment (81Denton) [inline code/datums/martial/cqc.dm]: >Culinary School Training

Why isn't this Close Quarters Cooking

Comment (Iamgoofball) [reply code/datums/martial/cqc.dm]: Holy shit

Comment (Ordonis) [reply code/datums/martial/cqc.dm]: Snake, try-a to remember the basics of-a CQC! I don't-a care what-a the crab tastes like, you have-a Greeks to fight!

**Comment by optimumtact** [comment]
I know this could use work, but we need to discourage people rolling chef

**Pull request merged**